index.html

```diff
@@ -68,25 +75,471 @@ <h3 class="main-subheading">
+      <div class="trending-products-container">
+        <div class="trending-product">
+          <figure class="trending-product-illustration">
+            <img src="#" alt="TrendingProduct" />
+            <img src="#" alt="TrendingBadge" />
+          </figure>
+          <div class="trending-product-details">
+            <a class="trending-product-title">Biztar</a>
+            <p class="trending-price">$12.00</p>
+            <ul class="trending-tags">
+              <li class="trending-tag">Business Consultant</li>
+              <li class="trending-tag">
+                <img src="#" alt="TagSpacer" class="trending-tag-spacer" />
+              </li>
+              <li class="trending-tag">UI Kits</li>
+            </ul>
+          </div>
+        </div>
+        <div class="trending-product">
+          <figure class="trending-product-illustration">
+            <img src="#" alt="TrendingProduct" />
+            <img src="#" alt="TrendingBadge" />
+          </figure>
+          <div class="trending-product-details">
+            <a class="trending-product-title">Auxell</a>
+            <p class="trending-price">$8.00</p>
+            <ul class="trending-tags">
+              <li class="trending-tag">Company Proposal</li>
+              <li class="trending-tag">
+                <img src="#" alt="TagSpacer" class="trending-tag-spacer" />
+              </li>
+              <li class="trending-tag">Print Template</li>
+            </ul>
+          </div>
+        </div>
+        <div class="trending-product">
+          <figure class="trending-product-illustration">
+            <img src="#" alt="TrendingProduct" />
+            <img src="#" alt="TrendingBadge" />
+          </figure>
+          <div class="trending-product-details">
+            <a class="trending-product-title">Phitaya</a>
+            <p class="trending-price">$10.00</p>
+            <ul class="trending-tags">
+              <li class="trending-tag">Handwritten Font</li>
+              <li class="trending-tag">
+                <img src="#" alt="TagSpacer" class="trending-tag-spacer" />
+              </li>
+              <li class="trending-tag">Fonts</li>
+            </ul>
+          </div>
+        </div>
+        <div class="trending-product">
+          <figure class="trending-product-illustration">
+            <img src="#" alt="TrendingProduct" />
+            <img src="#" alt="TrendingBadge" />
+          </figure>
+          <div class="trending-product-details">
+            <a class="trending-product-title">Rachelle</a>
+            <p class="trending-price">$16.00</p>
+            <ul class="trending-tags">
+              <li class="trending-tag">Media Kit</li>
+              <li class="trending-tag">
+                <img src="#" alt="TagSpacer" class="trending-tag-spacer" />
+              </li>
+              <li class="trending-tag">Fonts</li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div class="trending-ad-banner">
```

Comment:
Use the <aside> tag for content that is tangentially related to the main content. [Learn more about the <aside> element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside).